rules/windows/process_creation/proc_creation_win_code_devtunnel_tunneling.yaml

```
@@ -0,0 +1,37 @@
+title: Visual Studio Code/DevTunnel Abuse Detection
+id: 12345678-1234-5678-9123-456789abcdef
+description: |
+  Detects suspicious usage of Visual Studio Code's tunneling features, often abused in espionage campaigns.
+References:
+    - https://unit42.paloaltonetworks.com/stately-taurus-abuses-vscode-southeast-asian-espionage/
+status: experimental
+author: 0xAnalyst
+date: 2024/08/14
```

```suggestion
date: 2024-08-14
```
We've switched to the V2 specification